server/build/replace.js

```diff
@@ -7,10 +7,15 @@ export default async function replaceCurrentBuild (dir, buildDir) {
   const oldDir = join(buildDir, '.next.old')
 
   try {
-    await rename(_dir, oldDir)
+    await move(_dir, oldDir)
```

Comment:
Right, `mv` doesn't return a promise so that's why I created the `move` wrapper function and used that. Await works as expected.